### ISSUE TITLE
Change the project settings for our beePong to run on cluster PC (linux)

### ISSUE THREAD

**Issue by linhtng** [BeePong/42_transcendence#44]
I learnt from @liocle who confirmed it with titus and Markus that we don't actually need to run our project in a VM during eval.
Let change some setting so our project works on school PC without VM.
2 things I have in mind at least:
 - nginx port: change 443 to 1024, for example
 - CSRF_TRUSTED_ORIGINS: add "127.0.0.1"

If anything else, adapt and figure it out.

**Comment by linhtng** [comment]
When I tried running `make up_all ` in school PC, I got this error:
```
Error response from daemon: error while creating mount source path '/var/lib/docker/containers': mkdir /var/lib/docker/containers: permission denied
make: *** [Makefile:15: up_all] Error 1
```
which prevents Container beepong-filebeat to start. Can you please ask Markus if possible how to solve? @liocle 

Another error I got was:
```
Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error setting rlimits for ready process: error setting rlimit type 8: operation not permitted: unknown
make: *** [Makefile:15: up_all] Error 1
```
that prevented Container beepong-elasticsearch to start, but I applied the workaround I found from this [link](https://github.com/wazuh/wazuh-docker/issues/903), which is commenting out these lines:
```
# ulimits:
    #   memlock:
    #     soft: -1
    #     hard: -1
```
and it worked. But does that cause other issue, do you know?

**Comment by liocle** [comment]
Yes, it may be we have to remove the ulimits. Those, as far as i remember are needed to control the memory available to some services, but setting them may require elevated privileges, which this is the case. I think, Docker can handle this management by default ok considering we won't ever have too high load on any services ( except a bug!). I will check the other one with filebeat
